servicex/models.py

```diff
@@ -118,6 +134,7 @@ def accept(cls, email):
             raise NoResultFound(f"No user registered with email: {email}")
         pending_user.pending = False
         pending_user.save_to_db()
+        pending_user.send_email('emails/welcome.html')
```

Comment:
I think this is part of the workflow and should be in the `AcceptUser` resource

Reply:
Tricky because users can be accepted in two different places: via a POST to `/accept`, or via Slack, which needs its own dedicated endpoint (`/slack`) to handle all interaction.